apps/exalice/lib/exalice/storage.ex

```diff
@@ -0,0 +1,7 @@
+defmodule Storage do
+
+  @callback geocode(module :: atom, module :: atom, address :: String.t) :: Map.t
```

Comment:
It was intentional, however I will rename for the sake of clarity, thank you! :)